test/utilities/apexUtils.ts

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+    TextEditor
+} from 'wdio-vscode-service';
+import * as utilities from '../utilities';
+
+export async function createApexClassWithTest(name: string): Promise<void> {
+    const workbench = await browser.getWorkbench();
+    let textEditor: TextEditor;
+
+    // Using the Command palette, run SFDX: Create Apex Class to create the main class
+    const inputBox = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Apex Class', 1);
+
+    // Set the name of the new Apex Class
+    await inputBox.setText(name);
+    await inputBox.confirm();
+
+    // Select the default directory (press Enter/Return).
+    await inputBox.confirm();
+    await utilities.pause(1);
+
+    // Modify class content
+    const editorView = workbench.getEditorView();
+    textEditor = await editorView.openEditor(name + '.cls') as TextEditor;
+    await textEditor.setText('public with sharing class ' + name + ' {\n\tpublic static void SayHello(string name){\n\t\tSystem.debug(\'Hello, \' + name + \'!\');\n\t}\n}');
```

Comment:
Lines 31 and 49 have the code inline.  I kinda like the style/approach you have on lines 58-69, and lines 100-120, as they are more readable.  What are your thoughts on using that approach/style instead?

Reply:
Yeah, it's easier to read that way. I had forgotten to change it. Thanks!